src/Neo.VM/OpCode.cs

```diff
@@ -1973,7 +1973,7 @@ public enum OpCode : byte
         /// The item on top of main stack is removed and appended to the second item on top of the main stack.
         /// When we use this opcode, we should dup the second item on top of the main stack before using it.
         ///
-        /// <example> a a b -> a.concat(b)</example>
+        /// <example> a b -> a.concat(b)</example>
```

Comment:
To me it's not a typo because we have to `DUP` `a` in order to keep it on stack after `APPEND` finishes. `APPEND` does not push anything on stack in the end. See the summary for this method.

Reply:
Anna is correct, i meat to make it this way cause it is how this opcode works, just take a look at the above comment i made on this opcode.

Reply:
But, it depeends, i dont know where should we clearly describe it, cause i have no idea why this append opcode works this way, and much have a dup before calling it,,,,,,,,, maybe just add a little bit more explaination insead of simply update it.